packages/pyml/pyml.20210924/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer: "Thierry Martinez <martinez@nsup.org>"
+authors: "Thierry Martinez <martinez@nsup.org>"
+homepage: "http://github.com/thierry-martinez/pyml"
+bug-reports: "http://github.com/thierry-martinez/pyml/issues"
+license: "BSD-2-Clause"
+dev-repo: "git+https://github.com/thierry-martinez/pyml.git"
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+run-test: [make "test"]
+synopsis: "OCaml bindings for Python"
+description: "OCaml bindings for Python 2 and Python 3"
+depends: [
+  "ocaml" {>= "3.12.1"}
+  "dune" {build & >= "2.8.0"}
```

Comment:
The build tag must be stripped for dune.

Reply:
```suggestion
  "dune" {>= "2.8.0"}
```

Reply:
@thierry-martinez can you upstream the change?

Reply:
Done. Thanks! https://github.com/thierry-martinez/pyml/commit/44d49539e73be9562e6e74f64a15f72cc8c0cc57